Code update for inference

jeewonkimm2 opened the pull request:
I created a file called '[test_single_edit.py](https://github.com/jeewonkimm2/generative-inpainting-pytorch/blob/jeewon/test_single_edit.py)'.

- Overview : Enable inpainting at the coordinates you have specified.
1. Download checkpoint from (Trained PyTorch model: [[Google Drive](https://drive.google.com/open?id=1qbfA5BP9yzdTFFmiOTvYARUYgW1zwBBK)] [[Baidu Wangpan](https://pan.baidu.com/s/17HzpiqMPLIznvCWBfpNVGw)]) and save the entire folder called 'checkpoints'.
2. Input image size should be [256, 256].
3. Specify the coordinates(x1, y1, x2, y2) of inpainting part when you run the file.

```bash
python test_single_edit.py \
	--image examples/imagenet/imagenet_patches_ILSVRC2012_val_00008210_input.png \
	--output examples/output.png \
	--x1 100 \
	--y1 50 \
	--x2 180 \
	--y2 210
```
- --image : Path of an input image
- --output : Path of result
- x1, y1, x2, y2 : Coordinates

jeewonkimm2 committed:
Final Submit File

jeewonkimm2 committed:
Inference with setting customised coordinates

jeewonkimm2 committed:
Update codes

jeewonkimm2 committed:
Updated Readme file for customised use purpose

doorooful commented on .DS_Store:
Can you add this file format in gitignore?

doorooful commented on data/.DS_Store:
Same here!

doorooful commented on model/.DS_Store:
Same here!

doorooful commented on utils/.DS_Store:
Same here!

doorooful reviewed (commented):
Please update `.gitignore` file regarding mac.

jeewonkimm2 committed:
.gitignore for MacOS

jeewonkimm2 committed:
Added data/.gitignore, model/.gitignore, utils/.gitignore for MacOS

jeewonkimm2 commented:
I inserted them in `.gitignore`

```bash
.DS_Store
data/.DS_Store
model/.DS_Store
utils/.DS_Store
```

Can you check it please?

jeewonkimm2 committed:
Remove unnecessary code lines

doorooful reviewed (approved):
Thank you!

doorooful committed:
server setup

- Server installation & execution guide
- Input and Output functions with single image file(simply uploading and returning original file)

Signed-off-by: doorooful <codingdooroo@gmail.com>

jeewonkimm2 committed:
Merge pull request #1 from doorooful/server_setup

server setup

jeewonkimm2 committed:
Update README.md
